frontend/src/pages/RunDetailsV2.tsx

```diff
@@ -214,6 +230,23 @@ export function RunDetailsV2(props: RunDetailsV2Props) {
 
           // TODO(zijianjoy): Wait backend to supply run parameters, so UI can show them.
         )}
+
+        {/* Pipeline Spec tab */}
+        {selectedTab === 2 && isTemplatePullSuccessFromRun && (
+          <div className={commonCss.codeEditor} data-testid={'spec-ir'}>
+            <Editor
```

Comment:
NIT: I see that the `<Editor>` usage appears multiple places (PipelineDetailsV1/V2) page. To maintain consistency, it will be great to create a single component which wrap `<Editor>`. So PipelineDetail and RunDetail pages are consistent.